.github/workflows/retest.yaml

```
@@ -0,0 +1,24 @@
+name: Detect and Trigger Retest
+on:
+  issue_comment:
+    types: [created, edited]
+
+permissions:
+  contents: read
+
+jobs:
+  retest:
+    runs-on: ubuntu-latest
+    if: github.event.comment.author_association == 'MEMBER' && github.event.comment.body == '/retest'
+    permissions:
+      actions: write
+    steps:
+      - name: Retest
```

```suggestion
      - name: Re-run failed jobs for this PR
```

can be more specific here